Time results rewrite (#50)
greenelab/annorxiver_manuscript: content/04.results.md

The y-axis shows the number of days elapsed between when the first version of a preprint posted on bioRxiv and the time a preprint is published.
The color bar on the right represents the density of each hexbin in this plot: more dense regions are shown in a brighter color.
**B.** Preprints with more versions were associated with a longer time to publish.
The x-axis shows the number of versions of a preprint that were posted on bioRxiv.
The y-axis shows the number of days that elapsed between when the first version of a preprint was posted on bioRxiv and the date at which the peer reviewed publication appeared.
The density of observations are depicted with the violin plot with an embedded boxplot.
**C.** Author-selected categories were associated with modest differences in the time to publish.
Categories are shown on the y-axis.
The x-axis shows the median time-to-publish for each category.
Error bars represent 95% confidence intervals for each preprint category's median time to publication.
](https://raw.githubusercontent.com/danich1/annorxiver/e61dff8efe2216852a6b2a9ff0ec921a54356ad7/figure_generation/output/figure_four_panels.svg){#fig:publication_delay_panels width="100%"}

> The process of peer review includes a number of steps which take variable amounts of time [@doi:10.1002/nop2.51] and we examined if there is a difference in publication time between author-selected categories of preprints (Figure {@fig:publication_delay_panels}A).

Reviewer: ```suggestion
The process of peer review includes a number of steps which take variable amounts of time [@doi:10.1002/nop2.51].
To (DO X), we examined if there is a difference in publication time between author-selected categories of preprints (Figure {@fig:publication_delay_panels}A).
```
what is DO X here?

Of the most abundant preprint categories microbiology was the fastest to publish (140 days, (137, 145 days) [95% CI]) and genomics was the slowest (190 days, (185, 195 days) [95% CI]).

> We did observe category-specific differences; however, these differences were generally modest, suggesting that the peer review process overall is agnostic to preprint categories when delaying preprint publication time. 

Reviewer: ```suggestion
We did observe category-specific differences; however, these differences were generally modest, suggesting that the peer review process did not differ dramatically between preprint categories. 
```

One exception was the Scientific Communication and Education category, which took substantially longer to be peer reviewed and published (373 days, (373, 398 days) [95% CI]). 
This hints that there is a key step or steps within the peer review process that severely halts preprints in this category from being published.

> Preprint authors have the opportunity to update their preprint at any point until the peer reviewed publication appears. 

Reviewer: Claim structure

We found a positive correlation between preprints with multiple versions and time elapsed until publication (Figure {@fig:publication_delay_panels}B). 
Each new version adds additional 51 days before a preprint is published.
This time duration seems broadly compatible with the amount of time it would take to receive reviews and revise a manuscript, suggesting that many authors may be updating their preprints in response to peer reviews or other external feedback.

> We quantified the association between the magnitude of change to the text between the preprint and peer reviewed publication and the time to publication.

Reviewer: Claim structure?
